homework.md

```diff
@@ -44,70 +44,73 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language for the Web .
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+1.It is a good resources for developers to share code and work together.
+2.It's used for storing,tracking and collaborating on software projects.
+3.It is free.
+
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+You change the project without affecting the original repository.
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+You will get a copy of all the repository data.
```

Comment:
Maybe it's worth to mention that when you clone the repo --  you copy it from GitHub.com to your local machine?

Reply:
Thank you Alena .